crates/starknet/src/starknet/mod.rs

```diff
@@ -69,10 +71,16 @@ pub struct Starknet {
     pub(in crate::starknet) state: StarknetState,
     predeployed_accounts: PredeployedAccounts,
     pub(in crate::starknet) block_context: BlockContext,
-    blocks: StarknetBlocks,
+    // TODO: to avoid repeating some logic from blocks,
```

Comment:
I would remove todo but I would keep the comment.

Reply:
Fixed in e994a0fe9e906b077dcea7df34f9a74c854e815c.